dataland-backend/build.gradle.kts

```diff
@@ -1,3 +1,5 @@
+//dataland-backend
```

Comment:
braucht man den Kommentar hier?

Reply:
Ich glaube es ist hilfreich um sich in der IDE zurechtzufinden.